dist/tools/doccheck/exclude_patterns

```diff
@@ -10031,7 +10031,7 @@ drivers/include/sht2x\.h:[0-9]+: warning: Member SHT2X_USER_EOB_MASK \(macro def
 drivers/include/sht2x\.h:[0-9]+: warning: Member SHT2X_USER_HEATER_MASK \(macro definition\) of group drivers_sht2x is not documented\.
 drivers/include/sht2x\.h:[0-9]+: warning: Member SHT2X_USER_OTP_MASK \(macro definition\) of group drivers_sht2x is not documented\.
 drivers/include/sht2x\.h:[0-9]+: warning: Member SHT2X_USER_RESERVED_MASK \(macro definition\) of group drivers_sht2x is not documented\.
-drivers/include/shtc1\.h:[0-9]+: warning: unbalanced grouping commands
+drivers/include/shtcx\.h:[0-9]+: warning: unbalanced grouping commands
```

Comment:
Instead of adding this warning to the exclusion patterns, it should be fixed. Unlike the "Warning: Member ...", this can cause problems with the document structure.

Reply:
Done